### PR TITLE
#135842531 effective user landing page ui

### PR DESCRIPTION
**Pull request by collins-b**
#### What does this PR do?
Changes the look and feel of landing page to minimalistic design
 #### Description of Task to be completed?
Redesign the landing page for the game
 #### How should this be manually tested?
Run the project to observe the landing page
 #### Any background context you want to provide?
N/A
 #### What are the relevant pivotal tracker stories?
135842531
 #### Screenshots (if appropriate)
screen
how

 #### Questions:
N/A

### CODE REVIEW

File: app/views/includes/foot.pug [hunk 1]
```diff
@@ -8,6 +8,8 @@ script(type='text/javascript', src='/lib/underscore/underscore-min.js')
 
 //Bootstrap
 script(type='text/javascript', src='/lib/bootstrap/dist/js/bootstrap.js')
+script(type='text/javascript', src='/js/bootstrap.min.js')
```
Comment (collins-b) [inline app/views/includes/foot.pug]: Remove this because it is redundant

File: public/css/common.css [hunk 2]
```diff
@@ -1096,8 +870,6 @@ body {
...
   -moz-border-radius: 8px !important;
   border-radius: 8px !important;
-  text-transform: none !important;
-}
-.gradientButton .donateNow {
-  border-top: 0px solid black !important;
-}
+  text-transform: none !important; }
+  .gradientButton .donateNow {
+    border-top: 0px solid black !important; }
```
Comment (collins-b) [inline public/css/common.css:875]: I should edit the sass file.not this

**Review by collins-b — COMMENTED** [review]
Removed bootstrap.min.js

File: public/css/common.css (hunk 2 above)
Comment (collins-b) [reply public/css/common.css:875]: This is done

**Review by jackiewmacharia — APPROVED** [review]

**Comment by gngchrs** [comment]
@andela-cwekesa conflicts due to a merge. Please fix this.

**Pull request merged**